projecty/README.md

```diff
@@ -1,6 +1,8 @@
-Simple Logging Facade for Java (abbreviated SLF4J) – 
-acts as a facade for different logging frameworks
-It offers a generic API making the logging independent of the actual implementation.
-This allows for different logging frameworks to coexist. 
-It also helps migrate from one framework to another.
+The Model-View-Controller (MVC)
```

Comment:
Y this file changed ?

Reply:
I have just added the description whatever I did in my project